content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -30,7 +30,9 @@ getting killed by the kubelet before they are up and running.
 ## {{% heading "prerequisites" %}}
 
 
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+Kubernetes v1.16 and v1.17 require the `StartupProbe` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) to be set. Startup probes (as well as liveness and readiness probes) are enabled by default in Kubernetes v1.18 onwards.
```

Comment:
I don't think we need this paragraph. The simple fix is to omit `{{< version-check >}}`.

Reply:
You mean the entire fix is just to delete `{{< version-check >}}`? If so, done/updated. lmk if I understood correctly.